File: backend/src/models/entities/Questionario.ts
import {
    Entity,
    PrimaryGeneratedColumn,
    Column,
    OneToOne,
    CreateDateColumn,
} from "typeorm";
import { Avaliacao } from "./Avaliacao";

@Entity("questionarios")
export class Questionario {
    @PrimaryGeneratedColumn("uuid")
    id: string;

    @Column({ type: "int" })
    age: number;

    @Column({ type: "tinyint" })
    sex: number;

    @Column({ type: "tinyint" })
    chestPainType: number;

    @Column({ type: "float" })
    restingBloodPressure: number;

    @Column({ type: "float" })
    serumCholesterol: number;

    @Column({ type: "tinyint" })
    fastingBloodSugar: number;

    @Column({ type: "tinyint" })
    restingECG: number;

    @Column({ type: "float" })
    maxHeartRate: number;

    @Column({ type: "tinyint" })
    exerciseAngina: number;

    @Column({ type: "float" })
    oldpeak: number;

    @Column({ type: "tinyint" })
    stSlope: number;

    @CreateDateColumn({ type: "timestamp" })
    createdAt: Date;

    @OneToOne(() => Avaliacao, (avaliacao) => avaliacao.questionario)
    avaliacao: Avaliacao;
}

File: mobile/CardioCheck/Model/Questionario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Text.Json.Serialization;

namespace CardioCheck.Model
{
    public class Questionario
    {
        [JsonPropertyName("nome")]
        public string Nome { get; set; }

        [JsonPropertyName("age")]
        public int Age { get; set; }

        [JsonPropertyName("sex")]
        public int Sex { get; set; }

        [JsonPropertyName("chestPainType")]
        public int ChestPainType { get; set; }

        [JsonPropertyName("restingBloodPressure")]
        public float RestingBloodPressure { get; set; }

        [JsonPropertyName("serumCholesterol")]
        public float SerumCholesterol { get; set; }

        [JsonPropertyName("fastingBloodSugar")]
        public int FastingBloodSugar { get; set; }

        [JsonPropertyName("restingECG")]
        public int RestingECG { get; set; }

        [JsonPropertyName("maxHeartRate")]
        public float MaxHeartRate { get; set; }

        [JsonPropertyName("exerciseAngina")]
        public int ExerciseAngina { get; set; }

        [JsonPropertyName("oldpeak")]
        public float Oldpeak { get; set; }

        [JsonPropertyName("stSlope")]
        public int StSlope { get; set; }
    }
}
